src/pages/basic_data/institution_admin/person_register/PersonTableRow.tsx

```diff
@@ -165,7 +166,12 @@ export const PersonTableRow = ({
         dispatch(setNotification({ message: t('feedback.error.update_institution_user'), variant: 'error' }));
       }
     } else {
-      dispatch(setNotification({ message: t('feedback.error.update_person'), variant: 'error' }));
+      if (!cristinPerson.verified && isSuccessStatus(updateCristinPerson.status)) {
+        dispatch(setNotification({ message: t('feedback.success.update_person'), variant: 'success' })) &&
+          toggleDialog();
```

Comment:
hvorfor `&&`? Antar vi ønsker å gjøre begge
```suggestion
        dispatch(setNotification({ message: t('feedback.success.update_person'), variant: 'success' }));
        toggleDialog();
```